api.ts

```diff
@@ -2334,8 +2334,10 @@ export function makeSyncTable<
       throw new Error(`Sync table schemas with role ${TableRole.GroupMembers} must set a groupIdProperty`);
     }
 
-    if (!schemaDef.userIdProperty) {
-      throw new Error(`Sync table schemas with role ${TableRole.GroupMembers} must set a userIdProperty`);
+    if (!schemaDef.userIdProperty && !schemaDef.nestedGroupIdProperty) {
```

Comment:
is it valid to have a `nestedGroupIdProperty` without a `userIdProperty`? I'd think no... unless we let them have 2 GroupMembership role sync tables?

Reply:
You can have unlimited group membership role sync tables as of now.